middleware/rate_limiter.go

```diff
@@ -16,8 +16,8 @@ func RateLimiter(next http.Handler) http.Handler {
 		log.Fatal(err)
 	}
 	quota := throttled.RateQuota{
-		MaxRate:  throttled.PerMin(5),
-		MaxBurst: 5,
+		MaxRate:  throttled.PerMin(60),
+		MaxBurst: 60,
```

Comment:
This doesn't actually correspond to anything in ledger, right? There we just handle spikes by making the window 60 seconds long. (instead of say checking for 1 rps). I think 60 per minute is already quite generous - so maybe we just leave this 0?